R/drive_update.R

```diff
@@ -53,20 +57,10 @@ drive_update <- function(file = NULL,
   response <- make_request(request, encode = "json")
   proc_res <- process_response(response)
 
-  updated_file <- as_dribble(list(proc_res))
-  ## TO DO: this is a pretty weak test for success...
-  success <- proc_res$id == updated_file$id[1]
+  out <- as_dribble(list(proc_res))
 
-  if (success) {
-    if (verbose) {
-      message(
-        glue("\nFile updated with new media:\n  * {proc_res$name}\n",
-             "with id:\n  * {proc_res$id}")
-      )
-    }
-  } else {
-    stop("The file doesn't seem to have updated.", call. = FALSE)
+  if (verbose) {
+    message(glue("\nFile updated:\n  * {out$name}: {out$id}"))
   }
-
-  invisible(updated_file)
+  invisible(out)
 }
```

Comment:
I made this look more like its friends, e.g. `drive_copy()`. I think it's another case of "if we failed, we would know by now ... or, at least, this id check doesn't prove anything". Agree?